src/WaterPipe/WaterPipe.php

```diff
@@ -154,6 +154,14 @@ class WaterPipe
      */
     private $_mapRegistry;
 
+    /**
+     * In case an exception occures during pipe execution, it will be
```

Comment:
```suggestion
     * In case an exception occurs during pipe execution, it will be
```

Little typo here :-)